docs/Developer Reference/contributing-to-chainlink.md

```diff
@@ -0,0 +1,122 @@
+- [DEVEL]: The issue is a result of working on code found in the current repository
+- [FEAT]: The issue relates to a new feature request
+- [SMRT]: The issue related to using Chainlink smart contracts
+- [EXPL]: The issue related to using the Chainlink Explorer
+- [FAUC]: The issue related to using the Chainlink Faucet
+
+![Selecting the new issue category](/files/new-issue-category.png)
+
+After you select a category, enter the details for the issue. Include as much detail about the issue as possible. Provide a thorough description, environment, and software version details. Also provide detailed steps that describe how to reproduce the issue. The more thorough you make your description, the better the chances are that someone will be able to pick up the issue and resolve it.
+
+Once a team member acknowledges that the issue has been received, they will tag it with an appropriate label. You should then monitor the state of the open issue for any questions or updates.
+
+
+### Requesting a new Feature
+Have you thought of an improvement or an awesome new feature that you think should be implemented into Chainlink? Request a new feature to bring it to the attention of the team and the wider community. You can request new features by creating a new GitHub issue in the correct repository and tagging that issue with the [FEAT] prefix (Feature request). The process for doing this is covered in the [Raising an Issue](#raising-an-issue) section. Provide as much detail as possible in your feature request, including any benefits, risks, or considerations that you can think of.
+
+#### Voting on new features
+
+Sometimes a new feature is put to a vote to decide if it's something that the team and wider community should implement. When an feature is put to a vote, the issue is tagged with the 'needs votes' label. You can contribute to the voting process by reacting to the first post in the feature request with a thumbs up or thumbs down emoji. This will help drive the decision. You can also contribute your thoughts by replying directly to the feature request with a new post in the thread.
+
+![Voting on a new feature request](/files/voting-issue.png)
+
+### Submitting a Pull Request
+
+The best way to contribute to Chainlink is to submit a [pull request (PR)](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests). PRs can be submitted for various reasons, such as fixing an identified issue, adding a feature or improvement to the project, or even adding an entirely new repository to the Chainlink source code for a new tool or feature.
+
+If you're looking for something to pick up and create a PR for, you can search through the Chainlink repositories to find open issues, and approved feature requests. Alternatively, you can take a look at the current priorities for development listed on the Chainlink [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2129823).
```

Comment:
We don't actually use Pivotal anymore, and I don't think we publish the new alternative.

Reply:
thanks @alexroan. Removed that sentence